src/map/Pins.js

```diff
@@ -73,15 +91,34 @@
             })
             .append("title")
         ;
-        this.pinsPaths
-            .attr("stroke-width", 1.5 + "px")
+        gPinEnter
+            .append("text")
+            .attr("text-anchor","middle")
+        ;
+        this.pinsPaths.selectAll("text")
+            .style("stroke", this.fontColor())
+            .style("fill", this.fontColor())
+            .style("font-size", this.fontSize())
+            .style("font-family", this.fontFamily())
+            .style("dominant-baseline",this.textBaseline())
+            .attr("dx",0)
+            .attr("dy",context.pinTextDY())
+            .text(function(d){
+                return d[2] && d[2].text ? d[2].text : "";
+            });
+        this.pinsPaths.selectAll("path.data")
+            .attr("d", context.svgPinPath())
```

Comment:
Should be ```this.svgPinPath()``` - I think it would be worthwhile changing this to:
```
var svgPath =  this.svgPinPath();
this.pinsPaths.selectAll("path.data")
   .attr("d", svgPath)
...
```